### PR TITLE
gulp performance: add runtime pages

### PR DESCRIPTION
**Pull request by samouri**
Addresses https://github.com/ampproject/amphtml/issues/27955

### CODE REVIEW

**Comment by samouri** [comment]
@jridgewell  / @choumx: do you have any ideas for specific kinds of pages to throw in here?

**Comment by samouri** [comment]
@choumx: i've added an article page mostly based off of the pre-existing `examples/article.amp.html`. I've removed much of the styling as well as added the missing most popular amp components.

**Comment by choumx** [comment]
Looks like the example files were accidentally removed from the PR?

**Comment by samouri** [comment]
> Looks like the example files were accidentally removed from the PR?

Welp. Identified a bug in one of my main git flows. I usually use `git reset shaXYZ && git commit -a --amend` as a hack to quickly squash everything to an earlier commit. This made me realize it also untracks all new files. 

Just fixed it and pushed!

File: examples/runtime/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Runtime performance examples
+
+This folder contains example pages owned by [@wg-runtime](https://github.com/ampproject/wg-runtime), and made for testing performance with `gulp performance`
+
+- **images.html**: This page has 1000 images. It is a good stress test for the resources system as we want to ensure it prioritizes 1vp elements over other elements within the loadRect.
+- **list-always.html**: This page contains the slowest form of an `<amp-list>`, namely one with `binding="always"`.
```
Comment (choumx) [inline examples/runtime/README.md:6]: Looks like we're missing article.html here.

**Pull request merged**